frontend/src/app/coworking/widgets/room-reservation-table/room-reservation-table.widget.html

```diff
@@ -9,7 +9,10 @@
       </tr>
       @for (record of reservationsMap | keyvalue; track record.key) {
       <tr>
-        <th>{{ record.key }}</th>
+        <th>{{ record.key }}
+          <br><span class= "room-details"> {{"# seats: "}}{{ capacityMap[record.key]}}
+          <br>{{ capacityMap[record.key] === 2 ? "Pairing Room" : capacityMap[record.key] < 6 ? "Small Group" : "Large Group"}}</span>
```

Comment:
Nit: I think it may be good to move this naming to the backend! My fault, I should have caught this the first time - sorry about that. Ideally, the frontend should perform as little calculations as possible! Some recommendations:

- If these names are already stored in the backend, we can just bubble this up in the `RoomEntity`.

- If not, we can either:

  - Simply move this if statement / switch case to the backend when we return this data in the map.

  - We can also add a static method in the `RoomEntity` that returns the correct name based on the capacity (this can just be done in `.toModel()`!) This functionality could also be abstracted to an enumeration or could be added to a calculated field in the entity, though that is probably overkill here